software/arsnova.yml

```
@@ -0,0 +1,13 @@
+name: "ARSnova"
+website_url: "https://www.particify.de/en/"
+source_code_url: "https://gitlab.com/particify/dev/foss/arsnova-server"
+description: "Modern, web-based approach to Audience Response Systems (ARS). It is the basis for the online service Particify."
+licenses:
+  - MIT
```

```suggestion
  - GPL-3.0
```
As per comments above, changing the license from mit to gpl.